kornia/geometry/liegroup/se3.py

```diff
@@ -247,3 +247,22 @@ def inverse(self) -> 'Se3':
         """
         r_inv = self.r.inverse()
         return Se3(r_inv, r_inv * (-1 * self.t))
+
+    def adjoint(self) -> Tensor:
+        """Returns the adjoint matrix of shape :math:`(B, 6, 6)`.
+
+        Example:
+            >>> s = Se3.identity(1)
+            >>> s.adjoint()
+            tensor([[[1., 0., 0., 0., 0., 0.],
+                     [0., 1., 0., 0., 0., 0.],
+                     [0., 0., 1., 0., 0., 0.],
+                     [0., 0., 0., 1., 0., 0.],
+                     [0., 0., 0., 0., 1., 0.],
+                     [0., 0., 0., 0., 0., 1.]]], grad_fn=<CatBackward0>)
+        """
+        R = self.so3.matrix()
+        z = zeros_like(R)
+        row0 = concatenate((R, self.t * R), -1)
```

Comment:
```suggestion
        row0 = concatenate((R, So3.hat(self.t) * R), -1)
```
https://github.com/strasdat/Sophus/blob/23.04-beta/cpp/sophus/lie/se3.h#L146
and test: https://github.com/strasdat/Sophus/blob/23.04-beta/cpp/sophus/lie/se3.h#L139